### PR TITLE
fix(curriculum): improve a hint in the new JS RPG project

### LINKED ISSUE #48273: Sentence that needs improvement in the new JS rpg project
**Looking forward for reporting a security issue:**
@Dario-DC, reported sentence that needs more context, and I agree.

Here is the sentence

https://github.com/freeCodeCamp/freeCodeCamp/blob/a00844eac58a77f0a192a8443070a9699dbdb16e/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8f14fe6d1fc72454648c7.md#L26

I suggest

> Your function should start with `monsterStats.style.display = none`, don't add empty line after the function.

### PR DESCRIPTION
**Pull request by Rancho2002**
I have made the changes as requested i.e. https://github.com/freeCodeCamp/freeCodeCamp/issues/48273#issue-1425104016 to this line https://github.com/freeCodeCamp/freeCodeCamp/issues/48273#issuecomment-1294023664

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48273 

Testing In Gitpod:
image

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8f14fe6d1fc72454648c7.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ You should set `monsterStats.style.display` to `none`.
 assert.match(update.toString(), /monsterStats\.style\.display\s*=\s*('|")none\1/);
 ```
 
-This should be the first line of the `update` function.
+You should add your code in the first line of the ```update``` function.
```
Comment (Sboonny) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8f14fe6d1fc72454648c7.md:26]:
```suggestion
You should add your code in the first line of the `update` function.

**Review by Sboonny — COMMENTED** [review]
awesome work, sorry for misguiding you. you didn't need to use code snippet markdown, you can keep the previous `update`.

**Comment by Rancho2002** [comment]
> awesome work, sorry for misguiding you. you didn't need to use code snippet markdown, you can keep the previous `update`.

I committed again. You can check.

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Rancho2002, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**